client/src/cordova/apple/OutlineLib/VpnExtension/Sources/PacketTunnelProvider.swift

```diff
@@ -24,35 +25,71 @@ import Tun2socks
 @objcMembers
 public class SwiftBridge: NSObject {
 
-    /** Helper function that we can call from Objective-C. */
-    public static func getTunnelNetworkSettings() -> NEPacketTunnelNetworkSettings {
-      // The remote address is not required, but needs to be valid, or else you get a
-      // "Invalid NETunnelNetworkSettings tunnelRemoteAddress" error.
-      let settings = NEPacketTunnelNetworkSettings(tunnelRemoteAddress: "::")
+  /** Helper function that we can call from Objective-C. */
+  public static func getTunnelNetworkSettings() -> NEPacketTunnelNetworkSettings {
+    // The remote address is not required, but needs to be valid, or else you get a
+    // "Invalid NETunnelNetworkSettings tunnelRemoteAddress" error.
+    let settings = NEPacketTunnelNetworkSettings(tunnelRemoteAddress: "::")
 
-      // Configure VPN address and routing.
-      let vpnAddress = selectVpnAddress(interfaceAddresses: getNetworkInterfaceAddresses())
-      let ipv4Settings = NEIPv4Settings(addresses: [vpnAddress], subnetMasks: ["255.255.255.0"])
-      ipv4Settings.includedRoutes = [NEIPv4Route.default()]
-      ipv4Settings.excludedRoutes = getExcludedIpv4Routes()
-      settings.ipv4Settings = ipv4Settings
+    // Configure VPN address and routing.
+    let vpnAddress = selectVpnAddress(interfaceAddresses: getNetworkInterfaceAddresses())
+    let ipv4Settings = NEIPv4Settings(addresses: [vpnAddress], subnetMasks: ["255.255.255.0"])
+    ipv4Settings.includedRoutes = [NEIPv4Route.default()]
+    ipv4Settings.excludedRoutes = getExcludedIpv4Routes()
+    settings.ipv4Settings = ipv4Settings
 
-      // Configure with Cloudflare, Quad9, and OpenDNS resolver addresses.
-      settings.dnsSettings = NEDNSSettings(servers: ["1.1.1.1", "9.9.9.9", "208.67.222.222", "208.67.220.220"])
+    // Configure with Cloudflare, Quad9, and OpenDNS resolver addresses.
+    settings.dnsSettings = NEDNSSettings(servers: ["1.1.1.1", "9.9.9.9", "208.67.222.222", "208.67.220.220"])
 
-      return settings
+    return settings
+  }
+
+  /** Creates a new Outline Client based on the given transportConfig. */
+  public static func newClient(transportConfig: String) -> OutlineNewClientResult {
+    let result = OutlineNewClient(transportConfig)
```

Comment:
Don't you need to handle errors here?

Reply:
I handled it the next line (`result?.error != nil`), and now gomobile won't generate the `NSError` output because we are not returning Go's `error`.